commons/src/connections/contacts.ts

```diff
@@ -13,13 +13,22 @@ import type { MiddlewareResponse, Thread } from "@commons/types/Nylas";
 
 export const fetchContacts = async (
   query: ContactsQuery,
-  offset: number,
-  limit: number,
+  params: {
+    offset: number;
+    limit: number;
+    group?: string;
+  },
 ): Promise<Contact[]> => {
+  let url = `${getMiddlewareApiUrl(
+    query.component_id,
+  )}/contact-list/contacts?limit=${params.limit}&offset=${params.offset}`;
+
+  if (params.group) {
+    url = url + `&group=${params.group}`;
+  }
+
   const contacts = await fetch(
-    `${getMiddlewareApiUrl(
-      query.component_id,
-    )}/contact-list/contacts?limit=${limit}&offset=${offset}`,
+    url,
```

Comment:
I think this can be simplified if you remove `params: {...}` and change it `queryString: string` or `queryString: ContactSearchQuery`. That way it would look like this

```
  const contacts = await fetch(
    `${getMiddlewareApiUrl(
      query.component_id,
    )}/contact-list/contacts?${queryString}`,
```
and there wouldn't be conditional logic to change query params